### PR TITLE
Added Prime_L, Prime_O and Prime_M, moved Prime_R into primekb folder

### PR DESCRIPTION
**Pull request by MxBlu**
As the title describes, adding in the the Prime_* boards into "primekb".

### CODE REVIEW

File: keyboards/primekb/prime_r/readme.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Hardware Availability: [Prime_KB](https://www.primekb.com/)
 
 Make example for this keyboard (after setting up your build environment):
 
-    make prime_r:default
+    make primekb/prime_r:default
 
 See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
```
Comment (noroadsleft) [inline keyboards/primekb/prime_r/readme.md]: Can you update this line to the following?

`See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).`

Comment (MxBlu) [reply keyboards/primekb/prime_r/readme.md]: The code in the folder isn't actually maintained by me, but I guess it's not a bad thing to add in the link to the guide.  Updated.

**Comment by drashna** [comment]
While moving things around here ..... do any of these conform to these layouts? 
https://github.com/qmk/qmk_firmware/tree/master/layouts/default

If so, it may be worth changing the layout names, and adding `LAYOUTS` to the rules.mk for the boards: 
https://docs.qmk.fm/#/feature_layouts?id=supporting-a-layout

**Comment by MxBlu** [comment]
I've removed the `PREVENT_STUCK_MODIFIERS` define from the prime_r keyboard, so you can disregard the conflict.

**Comment by drashna** [comment]
Unfortunately, the conflict still exists, and we are unable to merge it until it's resolved.

**Comment by MxBlu** [comment]
Rebased on master, we should be good now.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**